src/index.ts

```diff
@@ -31,3 +31,6 @@ export const createApi = flow(initMakeRequest, makeRequest => ({
     getRelated: makeRequest(collections.getRelated),
   },
 }));
+
+export { Language, ColorId, ContentFilter, SearchOrderBy } from './methods/search/types';
+export { OrderBy, Orientation } from './types/request';
```

Comment:
Needed to make it easier for consumers to get the argument types without importing them from an obscure nested file.